src/utils/docs.jl

```diff
@@ -281,7 +281,7 @@ using Revise, Oscar;
 The first run of `build_doc` will take the usual few minutes, subsequent runs
 will be significantly faster.
```

Comment:
This docstring should also explain what the new kwargs do and which values are possible